Fix links and add two last awards in About page (#161)
openfisca/openfisca.org: content/about.en.md

OpenFisca was originally developed as a desktop application using the [Qt](https://www.qt.io/) library with a Python API. This source code was released under a free software (AGPL) license in November 2011.

In early 2014, the French Prime Minister task force for open data, [Etalab](https://www.etalab.gouv.fr/), started using OpenFisca and soon became a major contributor. After organising a [hackathon](https://www.eventbrite.fr/e/billets-hackathon-openfisca-10751826001) to give more visibility to the product, its team decided to:

<ul>
    <li>stop the development of the [Qt version](https://github.com/openfisca/openfisca-qt);</li>
    <li>separate the computing engine from its desktop user interface;</li>
    <li>offer a web API in addition to the Python API;</li>
    <li>demonstrate the value of the web API by developing sample applications including a web interface to simulate personal cases;</li>
    <li>offer public access to this web API.</li>

> </ul>

Author: @clementbiron This syntax doesn't fix the list. What would you recommend? ^^'

Reviewer: This file is in Markdown and should stay this way 🙂 The issue is not with the HTML, it is that default list styles have been reset and they should be styled.

In 2018, the product manager that had led the internationalisation efforts [moved](https://www.digital.govt.nz/blog/labplus-lessons-from-matti-schneider-about-service-innovation-at-french-state-incubator-beta-gouv-fr/) to Aotearoa New Zealand, where the recently established [Service Innovation Lab](https://serviceinnovationlab.github.io/projects/legislation-as-code/) was finishing writing its [Better Rules](https://www.digital.govt.nz/blog/what-is-better-rules/) report. This foundational theoretical piece matched perfectly with OpenFisca's implementation of “turning legislation into code”, leading to the Rates Rebate app, the first service in production outside of Europe. Other areas in the Pacific region (Australia, Polynesia, Canada) were progressively exposed to OpenFisca and adopted it.

> In 2019, OpenFisca gets the [most innovative open source softaware prize](https://joinup.ec.europa.eu/collection/sharing-and-reuse-it-solutions/sharing-reuse-awards-2019-results#oss-inno) from the Joinup program of the European Commission.  

Reviewer: ```suggestion
In 2019, OpenFisca is recognised as [most innovative open source software](https://joinup.ec.europa.eu/collection/sharing-and-reuse-it-solutions/sharing-reuse-awards-2019-results#oss-inno) by the European Commission through the Joinup program.
```

> This same year, the first [LexImpact](https://leximpact.an.fr) simulator is created. This impact assessment tool based on OpenFisca with the French National Assembly, enables members of parliament to evaluate reforms on test cases or the whole French population within a few seconds.

Reviewer: ```suggestion
On the same year, the first [LexImpact](https://leximpact.an.fr) simulator is created by the French Parliament, enabling elected representatives to evaluate reforms on both test cases and the whole French population within a few seconds.
```

Author: Thank you! Taking the update but keeping: 
* `French National Assembly` as its the [official name of the Assemblée nationale in English](https://www2.assemblee-nationale.fr/langues/welcome-to-the-english-website-of-the-french-national-assembly) while the `parliament` includes the Senate.
* `members of parliament` as LexImpact tools are available for members of the Senate that are elected by an indirect election while isn't `elected representatives` more for elections by citizens? 🤔

Reviewer: I just wanted to avoid the repetition Parliament / Members of Parliament 🙂 if you keep National Assembly, no problem for “members of parliament”.

In 2020, considering the growing adoption across administrations and the critical mass of contributors, the French Interministerial Direction of Digital Affairs (_DINUM_) decided to stop funding efforts to grow and maintain OpenFisca, suggesting these should be funded collectively by reusers. A minimal core team was maintained through funding from the French _Agence nationale de la cohésion des territoires (ANCT)_, but usage growth and technical improvements slowed down significantly as no clear business model was established.

In 2021, members of civil society started modelling British and then USA legislation, leading to the creation of [PolicyEngine](https://policyengine.org).

In 2022, as usage grew in several countries without clear contributions to an increasingly outdated engine, means for sustained operation of OpenFisca independently from any single State were assessed by the core team on the ANCT funds.

> In 2023, OpenFisca community gets the [Edge of Government Award](https://www.worldgovernmentsummit.org/awards/edge-of-government) at the World Government Summit.

Reviewer: ```suggestion
In 2023, the OpenFisca community is distinguished among over 1,000 candidates screened by the OECD and is awarded the [Edge of Government Innovation Award](https://www.worldgovernmentsummit.org/awards/edge-of-government) at the World Government Summit.
```
